tests/regression/28-race_reach/95-deref-fun.c

```diff
@@ -1,7 +1,7 @@
 // PARAM: --enable ana.sv-comp.enabled --set ana.specification "CHECK( init(main()), LTL(G ! call(reach_error())) )"
 #include<pthread.h>
 // NOCRASH
-void foo(int (*callback)()) {
+void foo(int (*callback)(void)) {
 }
```

Comment:
Since this was a nocrash test, I'm not sure if the modified test still meets the intention.

Reply:
Good point. It comes from #882, which looks like it shouldn't care about the arguments, it just boils down to `Cil.isFunctionType`.